make/variants-diagram.py

```
@@ -0,0 +1,329 @@
+                start_idx = i
+            elif (
+                start_idx != -1
+                and line.strip()
+                and not line.startswith("   ")
+                and not line.startswith("\t")
+            ):
+                end_idx = i
+                break
+
+        if start_idx == -1:
+            raise ValueError("Could not find graphviz section in web/src/variants.rst")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)